cogs/get_token_authorisation.py

```diff
@@ -19,90 +20,93 @@
             ".ASPXAUTH": settings["MEMBERS_LIST_URL_SESSION_COOKIE"],
         }
 
         REQUEST_URL: Final[str] = "https://guildofstudents.com/profile"
 
-        async with aiohttp.ClientSession(headers=request_headers, cookies=request_cookies) as http_session:  # noqa: E501, SIM117
+        async with aiohttp.ClientSession(headers=REQUEST_HEADERS, cookies=REQUEST_COOKIES) as http_session:  # noqa: E501, SIM117
             async with http_session.get(REQUEST_URL) as http_response:
                 response_html: str = await http_response.text()
 
-        TABLE_ID: str = "ulOrgs"
         parsed_html: bs4.Tag | bs4.NavigableString | None = BeautifulSoup(
             response_html,
             "html.parser",
-        ).find(
-            "ul",
-            {"id": TABLE_ID},
-        )
+        ).find("ul", {"id": "ulOrgs"})
 
         if parsed_html is None or isinstance(parsed_html, bs4.NavigableString):
-            NO_ADMIN_DEBUG: Final[str] = (
+            logger.debug(
                 "No admin table was found, meaning the token provided "
-                "does not have admin access to any societies."
+                "does not have admin access to any societies.",
             )
-            logger.debug(NO_ADMIN_DEBUG)
             await ctx.respond("The user token provided does not have any admin access!")
             return
 
-        orgs = [li.get_text(strip=True) for li in parsed_html.find_all("li")]
-
-        DIV_ID: Final[str] = "profile_main"
-        profile_section: bs4.Tag | bs4.NavigableString | None = BeautifulSoup(
+        profile_section_html: bs4.Tag | bs4.NavigableString | None = BeautifulSoup(
             response_html,
             "html.parser",
-        ).find(
-            "div",
-            {"id": DIV_ID},
-        )
+        ).find("div", {"id": "profile_main"})
 
-        if profile_section is None:
-            logger.warning("Couldn't find the profile of the user!")
-            logger.debug(response_html)
+        if profile_section_html is None:
+            logger.warning(
+                "Couldn't find the profile section of the user"
+                "when scraping the website's HTML!",
+            )
+            logger.debug("Retrieved HTML: %s", response_html)
             await ctx.respond(
                 "Couldn't find the profile of the user!"
                 "This should never happen, please check the logs!",
             )
             return
 
-        user_name: bs4.Tag | bs4.NavigableString | int | None = profile_section.find("h1")
+        user_name: bs4.Tag | bs4.NavigableString | int | None = profile_section_html.find("h1")
 
-        if type(user_name) is not bs4.Tag:
-            logger.debug("Found user profile but couldn't find their name!")
-            await ctx.respond("Found user profile but couldn't find the name!")
-            return
+        NO_PROFILE_DEBUG_MESSAGE: Final[str] = "Found user profile but couldn't find their name!"  # noqa: E501
```

Comment:
Define error message within `if` check to prevent declaring it if the error does not occur.